packages/connection-form/src/utils/csfle-handler.ts

```diff
@@ -213,7 +214,9 @@ export function textToEncryptedFieldConfig(text: string): Document | undefined {
 
   let parsed: Document = {};
   try {
-    parsed = parseFilter(text);
```

Comment:
Parse filter adds a layer of indirection, going to merge this so it bumps connection-form for vscode, feel free to revert if anything comes up.